README.md

```diff
@@ -1,18 +1,17 @@
-## Hourly News
+## News Skill
```

Comment:
I'm trying to avoid the word "Skill" in names.  We end up with "Skills list" containing "Abc Skill", "Foo Skill", "Bar Skill", ...

Maybe "News Feed"?